.travis.yml

```diff
@@ -0,0 +1,40 @@
+branches:
+  only:
+    - main
+
+language: rust
+rust:
+  - stable
+
+env:
+  global:
+    - SCCACHE_VERSION=0.2.13
+    - SCCACHE=$HOME/.sccache/bin/sccache
+    - RUSTC_WRAPPER=$SCCACHE
+
+cache:
+  directories:
+    - $HOME/.cache/sccache
+    - $HOME/.cargo
+    - $HOME/.rustup
+
+before_install:
+  - rustup component add clippy rustfmt
+  - cargo clippy --version
+  - cargo fmt --version
+  - |
+    mkdir -p $HOME/.sccache/bin
+    pushd $HOME/.sccache
+    curl -LO https://github.com/mozilla/sccache/releases/download/$SCCACHE_VERSION/sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl.tar.gz
+    tar -C bin -xvf sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl.tar.gz --wildcards '*/sccache' --strip 1
+    popd
+  - $SCCACHE --version
+
+script:
+  - cargo fmt -- --check
+  - cargo clippy --locked --workspace --all-features --all-targets -- -D warnings
+  - cargo build --locked --workspace --all-features --verbose
```

Comment:
Do we need the build at all? `cargo check` (which is part of clippy) already tells us whether the build would succeed so no need to actually do it, right?